Purchases/Public/RCPurchases.h

```diff
@@ -208,7 +208,7 @@ NS_SWIFT_NAME(Purchases)
  @param completion An optional completion block called when the aliasing has been successful. This completion block will receive an error if there's been one.
  */
 - (void)createAlias:(NSString *)alias completionBlock:(nullable RCReceivePurchaserInfoBlock)completion
-NS_SWIFT_NAME(createAlias(_:_:));
+NS_SWIFT_NAME(createAlias(_:_:)) __attribute((deprecated("Use logIn instead.")));
```

Comment:
Deprecated this one. We had it removed at one point, but I think it got re-added in some rebase. 
Since we're not doing a major, we should have it but deprecated.